imbi/endpoints/components/models.py

```diff
@@ -23,11 +23,11 @@ class ProjectComponentStatus(str, enum.Enum):
     FORBIDDEN = 'Forbidden'
 
 
-class ProjectStatus(str, enum.Enum):
+class ProjectStatus(int, enum.Enum):
     """Component Score project fact values"""
-    OKAY = 'Okay'
-    NEEDS_WORK = 'Needs Work'
-    UNACCEPTABLE = 'Unacceptable'
+    OKAY = 100
+    NEEDS_WORK = 80
+    UNACCEPTABLE = 0
```

Comment:
Why not make this `20`?

Reply:
Can't say that I really have a good reason for not making this 20... _commit incoming_

Reply:
I would now use this in the
```python
for max_value in (
    20,  # arbitrary low-end of middle range
````

Reply:
And `sorted(models.ProjectStatus)` is an even cleaner (and extensible) way to write it. See https://github.com/AWeber-Imbi/imbi-api/pull/107/commits/afd2b0a0884d37f5822399190b23162ce17cd758